en/controllers/components/csrf.rst

```diff
@@ -15,6 +15,12 @@ request data & cookie value. If either is missing or the two values mismatch the
 component will throw a
 :php:class:`Cake\\Network\\Exception\\InvalidCsrfTokenException`.
 
+.. note::
+    You should always verify the HTTP method being used before doing with
+    side-effects. You should :ref:`check the HTTP method <check-the-request>` or
```

Comment:
@markstory sorry to bother, that's probably due to my english skills but I don't totally get the idea behind: ``before doing with side-effects``.